local-notifications/ios/Plugin/LocalNotificationsDelegate.swift

```diff
@@ -0,0 +1,87 @@
+import Capacitor
+import UserNotifications
+
+public class LocalNotificationsDelegate: NSObject, NotificationHandlerProtocol {
```

Comment:
Similarly to push notifications, I think this class should be renamed since the term `delegate` can cause confusion.